File: src/main/java/Card/Blue/MultipleProcess.java
package Card.Blue;

import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
import com.megacrit.cardcrawl.cards.AbstractCard;
import com.megacrit.cardcrawl.characters.AbstractPlayer;
import com.megacrit.cardcrawl.core.CardCrawlGame;
import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
import com.megacrit.cardcrawl.localization.CardStrings;
import com.megacrit.cardcrawl.monsters.AbstractMonster;
import power.MultipleProcessPower;

public class MultipleProcess extends AbstractCard {
    public static final String ID = "Multiple Process";
    private static final CardStrings cardStrings;
    public static final String NAME;
    public static final String DESCRIPTION;
    private static final int COST = 1;



    public MultipleProcess() {
        super(ID, NAME, "1", COST, DESCRIPTION, CardType.POWER, CardColor.BLUE, CardRarity.UNCOMMON, CardTarget.SELF);
        this.baseMagicNumber = 3;
        this.magicNumber = baseMagicNumber;
    }



    public void use(AbstractPlayer player, AbstractMonster monster) {
        AbstractDungeon.actionManager.addToBottom(new ApplyPowerAction(player, player, new MultipleProcessPower(player, this.magicNumber), this.magicNumber));
    }

    public void upgrade() {
        if (!this.upgraded) {
            this.upgradeName();
            this.upgradeMagicNumber(1);
        }
    }

    public AbstractCard makeCopy() {
        return new MultipleProcess();
    }

    static {
        cardStrings = CardCrawlGame.languagePack.getCardStrings(ID);
        NAME = cardStrings.NAME;
        DESCRIPTION = cardStrings.DESCRIPTION;
    }
}

File: src/main/java/Card/Blue/MultipleProcess.kt
package Card.Blue

import com.megacrit.cardcrawl.actions.common.ApplyPowerAction
import com.megacrit.cardcrawl.cards.AbstractCard
import com.megacrit.cardcrawl.characters.AbstractPlayer
import com.megacrit.cardcrawl.core.CardCrawlGame
import com.megacrit.cardcrawl.dungeons.AbstractDungeon
import com.megacrit.cardcrawl.localization.CardStrings
import com.megacrit.cardcrawl.monsters.AbstractMonster
import power.MultipleProcessPower

class MultipleProcess : AbstractCard(ID, NAME, "1", COST, DESCRIPTION, CardType.POWER, CardColor.BLUE, CardRarity.UNCOMMON, CardTarget.SELF) {
    init {
        this.baseMagicNumber = 3
        this.magicNumber = baseMagicNumber
    }


    override fun use(player: AbstractPlayer, monster: AbstractMonster) {
        AbstractDungeon.actionManager.addToBottom(ApplyPowerAction(player, player, MultipleProcessPower(player, this.magicNumber), this.magicNumber))
    }

    override fun upgrade() {
        if (!this.upgraded) {
            this.upgradeName()
            this.upgradeMagicNumber(1)
        }
    }

    override fun makeCopy(): AbstractCard {
        return MultipleProcess()
    }

    companion object {
        const val ID = "Multiple Process"
        private val cardStrings: CardStrings
        val NAME: String
        val DESCRIPTION: String
        const  val COST = 1

        init {
            cardStrings = CardCrawlGame.languagePack.getCardStrings(ID)
            NAME = cardStrings.NAME
            DESCRIPTION = cardStrings.DESCRIPTION
        }
    }
}
